.github/workflows/deploy-enterprise.yml

```
@@ -51,7 +51,7 @@ jobs:
             domain: hyp3-tibet-jpl.asf.alaska.edu
             template_bucket: cf-templates-1or0efwqffkgd-us-west-2
             image_tag: latest
-            product_lifetime_in_days: 60
+            product_lifetime_in_days: 14
```

```suggestion
            product_lifetime_in_days: 7
```

Maybe 7 is more appropriate as charlie said "in the next week or so"